websites/migrations/0046_unpublish_fields.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 3.1.14 on 2022-04-22 16:05
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("websites", "0045_blank_publish_fields"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="website",
+            name="last_unpublished_by",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="unpublisher",
+                to=settings.AUTH_USER_MODEL,
+            ),
+        ),
+        migrations.AddField(
+            model_name="website",
+            name="unpublished",
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name="website",
+            name="unpublished_status",
```

Comment:
The other publishing status fields are

```
draft_publish_status
live_publish_status
```

What do you think of `unpublish_status`? To me `unpublish_status` sounds more event-related (or even `unpublishing_status`). :shrug: